spec/features/user_sees_pagination_for_reviews_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+feature "pagination" do
+  context "visiting index page with many reviews" do
+    let(:user) do
+      FactoryGirl.create(:user)
+    end
+
+    let(:piece) {
+      FactoryGirl.create(:piece) }
+
+    let!(:reviews) {
+      FactoryGirl.create_list(
+      :review, 100, 
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Indent the first parameter one step more than the start of the previous line.<br>Trailing whitespace detected.